parser/src/main/java/io/specmesh/apiparser/model/SchemaInfo.java

```diff
@@ -13,30 +13,21 @@ public class SchemaInfo {
     private final String schemaRef;
     private final String schemaFormat;
     private final String schemaIdLocation;
-    private String schemaIdPayloadLocation;
     private final String contentType;
     private String schemaLookupStrategy;
 
     /**
-     * @param schemaRef
-     *            location of schema
-     * @param schemaFormat
-     *            format of schema
-     * @param schemaIdLocation
-     *            ???
-     * @param schemaIdPayloadLocation
-     *            ???
-     * @param contentType
-     *            content type of schema
-     * @param schemaLookupStrategy
-     *            schema lookup strategy
+     * @param schemaRef            location of schema
+     * @param schemaFormat         format of schema
+     * @param schemaIdLocation     header || payload
```

Comment:
not something confluent have in theirs by the look of it.

Also, the depth of this property would make it common to both key and value, which seems wrong. 

Feels like this should be at the key/value level.  

Also, need a value of "none".   

Sounds like an enum.

Reply:
agree